core/src/main/java/org/geysermc/geyser/session/GeyserSession.java

```diff
@@ -1776,6 +1776,11 @@ public void sendAdventureSettings() {
         Collections.addAll(abilityLayer.getAbilitiesSet(), USED_ABILITIES);
 
         updateAbilitiesPacket.getAbilityLayers().add(abilityLayer);
+        // https://github.com/GeyserMC/Geyser/issues/3896 Add Base mode ability layer to fixing FOV locks when changed world
+        if (spectator) {
```

Comment:
Why not use the spectator check a few lines above, aka https://github.com/GeyserMC/Geyser/blob/master/core/src/main/java/org/geysermc/geyser/session/GeyserSession.java#L1763-L1772?
Additionally, is intended that spectator mode would have both the base and spectator ability layer?
See your own earlier issue/PR: https://github.com/GeyserMC/Geyser/issues/3769